asdf/yamlutil.py

```diff
@@ -405,6 +405,14 @@ def dump_tree(tree, fd, ctx, tree_finalizer=None, _serialization_context=None):
     yaml_version = tuple(
         int(x) for x in ctx.version_map['YAML_VERSION'].split('.'))
 
+    # add yaml %TAG definitions from extensions
+    for ext in _serialization_context._extensions_used:
+        ext_tags = getattr(ext.delegate, "yaml_tag_context", None)
```

Comment:
In the [YAML spec](https://yaml.org/spec/1.2/spec.html#tag/shorthand/) this is referred to as a "tag handle".  Can we call it that here too, instead of "context"?

Reply:
Also, this property should be added to both `Extension` and `ExtensionProxy` in `asdf/extension/_extension.py`.

Reply:
sure that sounds grood, I will look into it

Reply:
I am not sure how to correctly implement the `Extension` properties, if you would be so kind to take another look @eslavich